src/test/java/com/endava/soa_5/cucumber/hook/Hooks.java

```diff
@@ -1,19 +1,25 @@
 package com.endava.soa_5.cucumber.hook;
 
+import com.endava.soa_5.context.ScenarioContext;
 import io.cucumber.java.After;
 import io.cucumber.java.Before;
 
-public class Hooks {
+
+public class Hooks extends Base {
+    ScenarioContext context = ScenarioContext.getScenarioContext();
 
     @Before
     public void beforeScenario() {
         //log.info("-----------------START OF SCENARIO----------------");
         //TODO: uncomment when log is implemented
+        driver.get("http://demowebshop.tricentis.com/");
```

Comment:
Better to have such steps in scenario (can be background)

Reply:
Or even better - to read this url from properties. In case it will be changed - no need to search in project/feature files.